lib/pngstream.js

```diff
@@ -30,32 +31,37 @@ var Stream = require('stream').Stream;
  */
 
 var PNGStream = module.exports = function PNGStream(canvas, sync) {
```

Comment:
I would add a:

```js
if (!(this instanceof PNGStream)) {
  return new PNGStream(canvas, sync)
}
```

To avoid pollution of the global namespace if invoked without `new`.